src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Windows.cs

```diff
@@ -6,5 +6,16 @@ namespace System.Device.Gpio.Drivers
 {
     public partial class RaspberryPi3Driver : Windows10Driver
     {
+        protected ulong ClearRegister
+        {
+            get { throw new PlatformNotSupportedException(); }
```

Comment:
@tarekgh PTAL. Windows raspberry pi driver won't expose the device registers, so I thought that throwing would be the best option when this method is called on Windows. Do you agree?

Reply:
do you have any more information about the design of the pi support on Windows? maybe they can have something else.

---
In reply to: [276028609](https://github.com/dotnet/iot/pull/367#discussion_r276028609) [](ancestors = 276028609)

Reply:
Also, RGBLedMatrix project is marked as linux only but it got compiled with the rest of devices which is wrong to do 

---
In reply to: [276032568](https://github.com/dotnet/iot/pull/367#discussion_r276032568) [](ancestors = 276032568,276028609)

Reply:
> RGBLedMatrix project is marked as linux only

That is ok, that library will only get compiled once for linux and windows. The main library is the one that has 2 different implementations.